ivy/functional/ivy/experimental/elementwise.py

```diff
@@ -1358,3 +1358,40 @@ def binarizer(
     xc = ivy.copy_array(x, out=out)
     bin = ivy.where(xc > threshold, 1, 0)
     return bin
+
+
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+@handle_exceptions
+@handle_array_like_without_promotion
+def ldexp(
+    x: Union[ivy.Array, ivy.NativeArray],
+    /,
+    *,
+    i: Union[ivy.Array, ivy.NativeArray],
```

Comment:
I believe the second array input (whether you call it `i` or `x2`), should be positional-only and placed before the `/,` sign